eng/scripts/get_module_dirs.ps1

```diff
@@ -2,14 +2,20 @@ Param(
   [string] $serviceDir
 )
 
-$modDirs = [Collections.Generic.List[String]]@()
+function Get-ModuleDirs ([string] $serviceDir) {
+  $modDirs = [Collections.Generic.List[String]]@()
```

Comment:
Another nit, can we keep whitespacing consistent, this looks like two space tabs, but build.ps1 looks like four space tabs?

Reply:
The pre-existing ps1 code was all 2 spaces, but that is not my preferred spacing at all. Hey @benbp is there a standard spacing for powershell code?

Reply:
[This](https://poshcode.gitbook.io/powershell-practice-and-style/style-guide/code-layout-and-formatting#use-four-spaces-per-indentation-level) is the only source I can find. I also prefer four spaces in general.

Reply:
I'm using the built in code formatter on save in VSCode, and had some trouble convincing it to do what I wanted:

```json
        "editor.tabSize": 4,
        "editor.detectIndentation": false
```

did the job.